### ISSUE TITLE
timesteps

### ISSUE THREAD

**Issue by prinshul** [facebookresearch/BenchMARL#44]
Hi,

How timesteps are controlled for vmas?
I ran balance env with mappo with tuned hyperparameters. In wandb it shows 165 steps.
But when I plotted with  marl-eval I can see 1e7 timesteps.
![Screenshot 2024-01-16 at 8 54 10 PM](https://github.com/facebookresearch/BenchMARL/assets/13760669/c17bf6a9-e5e8-4e8b-9aad-a2047474bce2)
![bal](https://github.com/facebookresearch/BenchMARL/assets/13760669/cd94e602-6631-4490-b12b-2385cfc0a70d)

Also, the plot is slightly different from the one given for tuned MAPPO + Balance.

**Comment by matteobettini** [comment]
Hey! Thanks for reaching out.

So the steps you see in wandb above are training iterations.

The ones below are collected frames

 every training iteration you collect `on_policy_frames_per_batch` so if you multiply the x axis of the plot above by that you get the plot below.

Otherwise in wandb you can also change the x axis to `counters/total_frames` to obtain the plot below

<img width="730" alt="Screenshot 2024-01-16 at 15 30 03" src="https://github.com/facebookresearch/BenchMARL/assets/55539777/187fc6bd-e6a6-4a43-8098-82c307e2b86f">

Lemme know if this helps

**Comment by prinshul** [comment]
Got it. Thank you. 

Also, the plot is slightly different from the one given by you for tuned MAPPO + Balance
https://api.wandb.ai/links/matteobettini/r5744vas

![Screenshot 2024-01-16 at 9 09 58 PM](https://github.com/facebookresearch/BenchMARL/assets/13760669/35444a3f-69ba-4600-b618-d25baa98e170)

I got this:
![Screenshot 2024-01-16 at 9 15 13 PM](https://github.com/facebookresearch/BenchMARL/assets/13760669/a172fa81-edc0-4b39-82a9-8fd1f59a753f)


Although the mean reward is almost similar (but not the same though I just ran it without any change).
Is this due to randomness during multiple runs? Is this because of seed not set?
